e2e/tests/steps_init_test.go

```diff
@@ -45,6 +51,27 @@ type rootSuite struct {
 	// validator holds command results between runs and reports errors to the test suite
 	validator *validatorPod
 	// validatorA maps to suffix ID 001 of the kube pod that we use as our control agent
+
+	// servicerKeys is hydrated by the clientset with credentials for all servicers.
+	// servicerKeys maps servicer IDs to their private key as a hex string.
+	servicerKeys map[string]string
+
+	// appKeys is hydrated by the clientset with credentials for all apps.
+	// appKeys maps app IDs to their private key as a hex string.
+	appKeys map[string]string
+
+	// relaychains holds settings for all relaychains used in the tests
+	//	the map key is a constant selected as the identifier for the relaychain, e.g. "RelayChainETH" for Ethereum
```

Comment:
In the override we have the key `0001` pointing to the ETH gateway. Is that the equivalent of `RelayChainETH` here or do they represent different things?

Reply:
Yes, these are equivalent. I used `RelayChainETH` here as a more descriptive name, but can't do this in the override, as it needs to be consistent with the `chains` field of the servicer specified in the genesis.

Reply:
I didn't modify the genesis file to update the `chains` in the servicer section, as that may need a discussion of changing `chains` field itself to `services`.

Reply:
The simple solution to this is to simply say:

```
e.g. "RelayChainETH" represented as "0001" in other parts of the codebase for Ethereum
```

Just trying to put myself in the shoes of someone who is entering the codebase for the first time and feeling confused